bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,382 @@
+| warnings                         | general         | Warnings as integer value (0 - 2^32-1)                                                                                           | Number                   | R  | 
+| infos                            | general         | Infos as integer value (0 - 2^32-1)                                                                                              | Number                   | R  | 
+| noOfErrors                       | general         | Number of bit-coded errors (0 - 32)                                                                                              | Number                   | R  | 
+| noOfWarnings                     | general         | Number of bit-coded warnings (0 - 8)                                                                                             | Number                   | R  | 
+| noOfInfos                        | general         | Number of bit-coded infos (0 - 8)                                                                                                | Number                   | R  | 
+| errorsMsg                        | general         | Errors as string                                                                                                                 | String                   | R  | 
+| warningsMsg                      | general         | Warnings as string                                                                                                               | String                   | R  | 
+| infosMsg                         | general         | Infos as string                                                                                                                  | String                   | R  | 
+| statusFlags                      | general         | Status flags                                                                                                                     | String                   | R  | 
+| bypassStatus                     | general         | Status of the bypass (OFF = closed, ON = open)                                                                                   | Switch                   | R  | 
+| bypassFrom                       | unitConfig      | Bypass active from                                                                                                               | DateTime                 | RW | 
+| bypassTo                         | unitConfig      | Bypass active to                                                                                                                 | DateTime                 | RW | 
+
+
+Please also see `variables.json` for further details: https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/variables.json

```

Comment:
The url won't age well as I don't know if we keep the 2.5.x branch when we move to 3.0. Actually the link is already possble incorrect as the json file is in the wrong place and needs to be moved. But I have no other idea on how to do it better. So what value does this add to have a link to this page? Maybe just remove?

Reply:
Perfectly fine :)